admin_scripts/Dell/firmware_version_check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# SVCPLAN-5154: Dell firmware version check
+# Takes a node range as input, and uses racadm to list out hardware type, bios version and iDRAC version.
+
+racadmpath=/root/xcat-tools/admin_scripts/Dell/racadm.sh
+
+# If user does not provide a node range, prompt them for one here. Invalid entries will cause the script to run against all nodes.
+if [ -z "$1" ]; then
+  read -p "Please enter a node range: " RANGE
+else
+  RANGE=$1
+fi
+
+# Assign the node range into an array to loop through
+readarray -t ARRAY < <(nodels $RANGE)
+
+# Print table headers with formatting
+printf "Running scan on noderange: $RANGE \n"
+printf "%15s %10s %-15s %-19s %-20s \n" "Nodename" ""  "Bios" "iDRAC" "Model"
```

Comment:
Another improvement would be to dynamically size the width of the Nodename field to the width of the longest node name returned by `nodels`. I can imagine that at somepoint we will hit hostnames that are longer than 15 characters.